### DISCUSSION TITLE
Change background color of picker component

### DISCUSSION THREAD

**Discussion by ip883** [wix/react-native-ui-lib#2218] [Q&A]
Hi,
I would like to change the background color of the modal picker dialog (<Picker>) to black. 

I already tried 

`
ThemeManager.setComponentForcedTheme('Picker', (props, context) => {
      return {
        backgroundColor: 'black',
      };
    });
`

and 

`
ThemeManager.setComponentForcedTheme('Dialog', (props, context) => {
      return {
        backgroundColor: 'black',
      };
    });
`

but it didn't change the background color of the picker where I can choose between the different options. How would I change the color?

Thanks!

![Picker Component](https://user-images.githubusercontent.com/78526960/186346558-66de1dff-02ce-40b5-8339-2b5091038956.png)

**Comment by matjua17** [comment]
Hi, 

You can customize the picker component through `listProps` and `topBarProps` Props. 

topBarProps: https://wix.github.io/react-native-ui-lib/docs/components/overlays/Modal/Modal.TopBar
listProps: `FlatListProps`

Code snippet:
```
<Picker
  topBarProps={{
    containerStyle: { backgroundColor: 'black' },
    titleStyle: { color: 'white' },
    cancelButtonProps: { iconStyle: { tintColor: 'white' } },
  }}
  listProps={{
    style: { backgroundColor: 'black' },
  }}
>
{...}
</Picker>
```

**Reply by LazyAfternoons** [reply]
Thank you @matjua17, what about the useWheelPicker prop though? How can I customize that modal and specifically the color of the highlighted list item which is purple by default? 


Thanks.
![Screenshot 2022-09-19 at 15 12 08](https://user-images.githubusercontent.com/12371664/191025210-30895a0a-09a2-4285-91b5-711ba7d73812.png)

EDIT: By looking at the [source code](https://github.com/wix/react-native-ui-lib/blob/aaaf4c37407b2db966bc6ce2baf065394a4145ff/src/components/picker/PickerItemsList.tsx#L81) it doesn't seem like any style is applied when rendering the wheel, except for the title and the doneLabel properties from topBarProps.

**Reply by matjua17** [reply]
Yes, unfortunately no style props are passed to the wheelpicker component when using `useWheelPicker` prop.

**Reply by LazyAfternoons** [reply]
> Yes, unfortunately no style props are passed to the wheelpicker component when using `useWheelPicker` prop.

I see, thank you.

**Comment by jjjkkkjjj** [comment]
NOTE

You can also change whole background color  by setting `overlayBackgroundColor`  in `modalProps `

```typescript
<Picker
    ...
    customPickerProps={{
                  modalProps: {
                    overlayBackgroundColor: 'black'
                  },
                }}
/>
```
